internal/provider/incident_workflow_resource.go

```diff
@@ -79,7 +126,7 @@ func (r *IncidentWorkflowResource) Create(ctx context.Context, req resource.Crea
 			Name:             data.Name.ValueString(),
 			TerraformRepoUrl: data.TerraformRepoURL.ValueStringPointer(),
 			OnceFor:          []string{"incident.url"},
-			ConditionGroups:  []client.ExpressionFilterOptsPayloadV2{},
+			ConditionGroups:  toPayloadConditionGrouos(data.ConditionGroups),
```

Comment:
```suggestion
			ConditionGroups:  toPayloadConditionGroups(data.ConditionGroups),
```

Reply:
!! the number of times I read this yesterday 😂